simtbx/diffBragg/src/diffBragg.cpp

```diff
@@ -382,6 +382,10 @@ diffBragg::diffBragg(const dxtbx::model::Detector& detector, const dxtbx::model:
                        0,.16,0,
                        0,0,.16;
 
+    db_cryst.rotate_principal_axes << 0.70710678,-0.70710678,0.,
```

Comment:
Add a comment talking about justification for this default in terms of the calmodulin paper
Also change the root2 to make it computed as sqrt(2) in double precision